MemoryLeak.Dockerfile

```
@@ -0,0 +1,32 @@
+FROM node:23-alpine3.20
+
+RUN apk add --no-cache \
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)